tests/testCheckerMain.ml

```diff
@@ -43,6 +43,21 @@ let set_last_price_in_wrapper wrapper price_option =
       {wrapper with deployment_state = Sealed {state with last_price = price_option}}
   )
 
+let debug_print_all_kit_in_sealed_state msg wrapper =
```

Comment:
This might actually be something useful to drop into `testLib.ml` in case we need to do similar debugging in the future.

Reply:
Ah, right, good point. Move it :+1: